app/Database/migrations/2018_05_16_200138_change_route_leg_to_unsigned_integer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ChangeRouteLegToUnsignedInteger extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('flights', function (Blueprint $table) {
+            $table->unsignedInteger('route_leg')->nullable()->change();
```

Comment:
I'll move this into the main migration. I'm requiring re-installs for right now since there would be too many migrations